src/common/utils/sendMessage.js

```diff
@@ -0,0 +1,58 @@
+import { ipcRenderer } from 'electron';
+
+const requestMap = {};
+const typeHandlerMap = {};
+
+window.addEventListener('__RENDERER_MESSAGE__', event => {
+  ipcRenderer.send('__RENDERER_MESSAGE__', event.detail);
+});
+
+ipcRenderer.on('__MAIN_MESSAGE__', (event, data) => {
+  window.dispatchEvent(new CustomEvent('__MAIN_MESSAGE__', { detail: data }));
+});
+
+window.addEventListener('__MAIN_MESSAGE__', event => {
```

Comment:
Codacy found an issue: [A function should not mix return statements with and without a result.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=6279215)